src/central_control/scripts/central_control.py

```diff
@@ -25,23 +25,25 @@ def listen(self):
         rospy.Subscriber("braking", Float64, self.handle_breaking_data)
         rospy.Subscriber("throttling", Float64, self.handle_throttling_data)
         rospy.Subscriber("sign_detection", DetectionResult, self.handle_sign_recognition)
-        rospy.spin()
+
+        rate = rospy.Rate(30)
```

Comment:
Forgot to use "rate.sleep" here in the while loop.

Reply:
yup woop my bad, second time I have done this I think